src/main/java/com/example/demo/service/BattleService.java

```diff
@@ -41,11 +47,23 @@ private List<Battle> findBattlesEndWithinPerm(int perm) {
 		return battleRepository.findByStatusAndUpdatedAtBetween(BattleStatus.END, lastBattleDay, now);
 	}
 
-	public BattleDetailsListResponseDto getBattleDetailsListInProgress() {
+	public BattleDetailsListResponseDto getBattleDetailsListInProgress(Principal principal) {
+		Member member = principalService.getMemberByPrincipal(principal);
 		List<Battle> battleListInProgress = battleRepository.findAllByStatusEquals(BattleStatus.PROGRESS);
+		List<Vote> votes = voteRepository.findAllByVoterId(member.getId());
+
+		List<Battle> notVotedBattleListInProgress = battleListInProgress.stream()
```

Comment:
notVotedBattleList하고 하니까 사용자가 투표하지 않은 배틀 리스트가 아니라 그냥 투표가 되지 않는 배틀 리스트를 가져오는 것 같은 느낌이 듭니다. 
```suggestion
		List<Battle> battleListInProgressNotVotedbyMember = battleListInProgress.stream()
```
같은 느낌은 어떤가요